src/components/deductions/ItemizedDeductions.tsx

```diff
@@ -230,3 +230,5 @@ export default function ItemizedDeductionsInfo(): ReactElement {
     </form>
   )
 }
+
+export default ItemizedDeductions
```

Comment:
@zakpatterson, should this be `ItemizedDeductionsInfo`?

Reply:
Yep! oops.

Reply:
I update it.